### ISSUE TITLE
release now fails because of PEP440

### ISSUE THREAD

**Issue by sfermigier** [zestsoftware/zest.releaser#68]
Trying to release a piece of software, I get the following error:

```
stefanesmacbook% release
Tag needed to proceed, you can use the following command:
git tag 0.2.1.dev0.dev190.g98425aa
/usr/local/lib/python2.7/site-packages/setuptools/dist.py:298:UserWarning:Theversionspecified('0.2.1.dev0.dev190.g98425aa')isaninvalidversion,thismaynotworkasexpectedwithnewerversionsofsetuptools,pip,andPyPI.PleaseseePEP440formoredetails.
"details."%self.metadata.version -m "Tagging 0.2.1.dev0.dev190.g98425aa
/usr/local/lib/python2.7/site-packages/setuptools/dist.py:298:UserWarning:Theversionspecified('0.2.1.dev0.dev190.g98425aa')isaninvalidversion,thismaynotworkasexpectedwithnewerversionsofsetuptools,pip,andPyPI.PleaseseePEP440formoredetails.
"details."%self.metadata.version"
Run this command (Y/n)?
/bin/sh: -c: line 1: syntax error near unexpected token `'0.2.1.dev0.dev190.g98425aa''
/bin/sh: -c: line 1: `/usr/local/lib/python2.7/site-packages/setuptools/dist.py:298:UserWarning:Theversionspecified('0.2.1.dev0.dev190.g98425aa')isaninvalidversion,thismaynotworkasexpectedwithnewerversionsofsetuptools,pip,andPyPI.PleaseseePEP440formoredetails.'


Failed to create tag 0.2.1.dev0.dev190.g98425aa
/usr/local/lib/python2.7/site-packages/setuptools/dist.py:298:UserWarning:Theversionspecified('0.2.1.dev0.dev190.g98425aa')isaninvalidversion,thismaynotworkasexpectedwithnewerversionsofsetuptools,pip,andPyPI.PleaseseePEP440formoredetails.
"details."%self.metadata.version!
```

(`prerelease` also prints similar messages, but only as a warning.)

I suspect this is related to the fact that I have installed recently `setuptools` 8.2.1.

If I revert `setuptools` to, say, version 3.6, it works as before.

**Comment by reinout** [comment]
I released 3.55. Can you check if that works for you? There's some error output suppression in there related to `setup.py`.

**Comment by reinout** [comment]
I'm assuming it works. Please re-open the ticket if it still fails for you, ok?
